oem/firstvoices/ios/FirstVoices/ViewController.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  ViewController.swift
+//  FirstVoices app
+//
+//  License: MIT
+//
+//  Copyright © 2019 FirstVoices.
+//
+//  Created by Serkan Kurt on 17/11/2015.
+//  Converted and rewritten by Marc Durdin on 15/05/2019.
+//
+
+import UIKit
+import KeymanEngine
+
+class ViewController: UIViewController, UIWebViewDelegate {
+
+  @IBOutlet var webView: UIWebView!
+  @IBOutlet var statusBarBackground: UIView!
+
+  override func viewDidLoad() {
+    super.viewDidLoad()
+
+    //
+    // Install all the .js files into Keyman Engine (doesn't show them in the keyboard list)
+    //
+
+    let kbPaths: [String] = Bundle.main.paths(forResourcesOfType: FVConstants.keyboardType,
+                                              inDirectory: FVConstants.keyboardsPath)
+    for kbPath in kbPaths {
+      let pathUrl = URL(fileURLWithPath: kbPath)
+      let id = pathUrl.deletingPathExtension().lastPathComponent
+      do {
+        try Manager.shared.preloadFiles(forKeyboardID: id, at: [pathUrl], shouldOverwrite: true)
+      } catch {
+        print("Failed to preload "+id+": "+error.localizedDescription)
+      }
+    }
+
+    //
+    // Show Instructions page
+    //
+
+    self.webView!.delegate = self
+    let filePath: String? = Bundle.main.path(forResource: FVConstants.instructionsName,
+                                             ofType: FVConstants.instructionsType,
+                                             inDirectory: FVConstants.instructionsPath)
```

Comment:
Making sure I understand this class - this presents an instructional WebView for new users?

Reply:
There's basically a single view which has instructions and a link to keyboard setup.